packages/linter/src/cli.ts

```diff
@@ -0,0 +1,238 @@
+import { readFileSync } from "fs";
```

Comment:
There's already an existing CLI, would it make sense to integrate the linter there?

Reply:
Added simple support in a5b25d71af56bff61f1e4f9ee4e5052603ac0b11. (Only single URL, to do more I need to merge option parsing mechanisms.)